### DISCUSSION TITLE
Error using  neuralcoref

### DISCUSSION THREAD

**Discussion by JessitaMS** [explosion/spaCy#10462] [Help: Installation]
When the neuralcoref extension is used in spacy, execution is stopped and the kernel is restarted, and of course the coref_clusters can never be executed. How can this wonderful function be used without this problem happening?

**Comment by polm** [comment]
I'm not sure about this specific problem, but Neuralcoref only works with spaCy v2, and isn't currently maintained. We're working on a new coref component, you can keep up with progress at #7264. 

Are you using spaCy v2? What is your code?

**Reply by JessitaMS** [reply]
Well, now I don´t have that error.

Yes, I'm using spaCy v2. I clone the neuralcoref git repository:
![imagen](https://user-images.githubusercontent.com/26332548/157848089-2120cada-615a-4c52-90bf-22b55bb89d8f.png)

And trying to install the requirements:
![imagen](https://user-images.githubusercontent.com/26332548/157848158-206928f6-838a-41a1-8f6d-831ad2a86e28.png)

but fail:
![imagen](https://user-images.githubusercontent.com/26332548/157868259-88dada03-e46e-4e22-ba58-365ecde23948.png)

**Reply by polm** [reply]
Please don't post screenshots of text, copy and paste the text. 

It's not possible to troubleshoot this based on the output you have provided. You would need to share the logs, or if you scroll up in your terminal before the `ERROR` you should see some kind of output with more details.

Note that our ability to provide support for spaCy v2, and especially neuralcoref, is limited. You might consider [coreferee](https://github.com/msg-systems/coreferee), which is actively developed, instead.

**Comment by svlandeg** [comment]
Update: our coref component for spaCy v3 is now released in `spacy-experimental`, cf https://github.com/explosion/spaCy/discussions/11585!
